scripts/observability/stackpack.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+#######################################
+# Delete a StackPack instance from SUSE Observability
+# Arguments:
+#   url (SUSE Observability)
+#   service_token (SUSE Observability)
+#   cluster_name
+# Examples:
+#   observability_delete_stackpack https://obs.suse.com/ xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx demo
+#######################################
+observability_delete_stackpack() {
+  local url=$1
+  local service_token=$2
+  local cluster_name=$3
+
+  local stackpacks=$(/usr/local/bin/sts stackpack list-instances --name kubernetes-v2 -o json --url $url --service-token $service_token)
```

Comment:
I'm not familiar with /usr/local/bin/sts, do we have something to check it's installed on the machine?